src/components/info_block.js

```diff
@@ -55,13 +52,11 @@ InfoBlock.propTypes = {
 };
 
 InfoBlock.defaultProps = {
-  heading: "Heading example",
-  description:
-    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
-  buttonText: "Example Button Text",
+  heading: "",
+  description: "",
+  buttonText: "",
```

Comment:
Should we have a default image as well? Or at least handle when one isn't provided